apps/src/netsim/NetSimLobby.js

```diff
@@ -508,7 +508,7 @@ NetSimLobby.prototype.onWireTableChange_ = function () {
 NetSimLobby.prototype.getUserSections_ = function (callback) {
   var memberSectionsRequest = $.ajax({
     dataType: 'json',
-    url: '/v2/sections/membership'
+    url: '/dashboardapi/sections/membership'
```

Comment:
once you have done all the other stuff recommended below: assuming NetSim is always served from dashboard (which @islemaster could perhaps confirm), you should change this url to hit `/api/v1/sections/membership`. The best practice is to only use the `dashboardapi` routes from pegasus, in part to help us "count down" the number of app-like things in pegasus which are reaching over into dashboard APIs.